FlightReservation/components/loginView.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+} from 'react-native';
+import SignUpButton from './signUpButtons';
+import Form from './formComponent';
+import ModalCustom from './modalComponent';
+
+function LoginView(props) {
+  const {signUpGoogle,validEmail,setValidEmail, validPassword, setValidPassword, setIsLoginFormActive,isLoginFormActive,showObj, showModal, addFill, formObjectState, modalVisible, singedText, isIconCheck} = props;
```

Comment:
Add prettier in this line.
You can do it online. https://prettier.io/

Reply:
Done!